src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js

```diff
@@ -1,18 +1,18 @@
 import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
-import { PROFILE_PATHS } from '../../constants';
+import { PROFILE_PATHS } from '@@profile/constants';
 
-import mockUserNotInEVSS from '../fixtures/users/user-non-vet.json';
-import mockUserInEVSS from '../fixtures/users/user-36.json';
+import mockUserNotInEVSS from '@@profile/tests/fixtures/users/user-non-vet.json';
```

Comment:
I'm less sure if `@@profile/tests` is better than `../../`. Thoughts?

Reply:
Definitely better IMO to use the alias.